docs/spec.md

```diff
@@ -1136,23 +1138,27 @@ def batch_norm_inference(operand, scale, offset, mean, variance, epsilon, featur
   return add(multiply(scale_bcast, normalized_operand), offset_bcast)
 ```
 
+For quantized types, performs
+`quantize(batch_norm_inference(dequantize(operand), scale, offset, mean,
+            variance, epsilon, feature_index), type(result))`.
```

Comment:
Hmm can't say that this definitions looks super satisfying. Why are we quantizing just the operand? Also, C2 doesn't seem to work under the current proposal.

Reply:
Not considering the other operands for quantized type was an oversight. Fixed it. PTAL.

I note that:

Following how the `scale, offset, mean, variance` are used in the spec as part of `broadcast_in_dim` op and from the constraint `C`1` of [broadcast_in_dim](https://github.com/openxla/stablehlo/blob/main/docs/spec.md#broadcast_in_dim), these other operands must be of quantized type if `operand` is of quantized type.  This also follows from the general strategy tahat have been employed so far while defining the quantized spec - all the operands with floating point types can potentially be considered with quantized type. 



Please note that the program constant operand `epsilon` is not consider to be of quantized type. IMO, considering that operand with quantized type unnecessary complicate the expressiveness of the op.